### PR TITLE
New default branch

### PR DESCRIPTION
**Pull request by ashleypean**
# Overview
- Updated the GraphQL schema to send a successful request
- Added mongoose schemas

## Updates
- Added GraphQL request and resolvers to backend 

## Demo
N/A

## How to Test
Not testable yet

## Remaining Works
- Add remaining queries on backend so that frontend can use them to test

## Related Tickets/Stories on Trello
N/A

### CODE REVIEW

**Review by dylan2040 — APPROVED** [review]
nice work you amazing person! :)

**Review by dylan2040 — APPROVED** [review]
did i say great work? great work!

File: client/src/Components/RenteeProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+function RenteeProfile (){
+
+}
```
Comment (htinlinnag1993) [inline client/src/Components/RenteeProfile.tsx:5]: Is this the add-on in the next PR?

**Review by htinlinnag1993 — APPROVED** [review]
Looks great! Ready to merge. May be remove the dead codes and comments.

**Pull request merged**